e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ShellAO.java

```diff
@@ -81,6 +82,11 @@ public ShellAO assertOutputDoesNotContain(String... messages) {
         return this;
     }
 
+    public ShellAO assertPageContainsString(String str) {
+        assertTrue(context().text().contains(str));
+        return this;
+    }
```

Comment:
Can we use `assertPageContains(String text)` method instead of this? It looks like they do the same thing